packages/main/src/Input.js

```diff
@@ -1080,6 +1084,16 @@ class Input extends UI5Element {
 		};
 	}
 
+	get inputAttributes() {
+		return {
+			"input": {
```

Comment:
The `input` key is redundant, we'll never want to modify any other tag except for the native input in such a way.

Reply:
Done, removed